Compile fail on Tri-State

Attempting to update Tri-state after a core dump gives:

```
   [CCi] chan_simpleusb.i -> chan_simpleusb.o
chan_simpleusb.c: In function ‘hidthread’:
chan_simpleusb.c:1110:21: error: variable ‘checksum’ set but not used [-Werror=unused-but-set-variable]
 1110 |      unsigned short checksum;
      |                     ^~~~~~~~
cc1: all warnings being treated as errors
make[1]: *** [/usr/src/asterisk-20.2.1/Makefile.rules:150: chan_simpleusb.o] Error 1
make: *** [Makefile:396: channels] Error 2
```

--- a/channels/chan_simpleusb.c
+++ b/channels/chan_simpleusb.c
@@ -1107,9 +1107,7 @@ static void *hidthread(void *arg)
 			if (o->wanteeprom) {
 				ast_mutex_lock(&o->eepromlock);
 				if (o->eepromctl == 1) {	/* to read */
-					unsigned short checksum;
 					/* if CS okay */
-					checksum = ast_radio_get_eeprom(usb_handle, o->eeprom);
 					if (!ast_radio_get_eeprom(usb_handle, o->eeprom)) {
 						if (o->eeprom[EEPROM_USER_MAGIC_ADDR] != EEPROM_MAGIC) {
 							ast_log(LOG_ERROR, "Channel %s: EEPROM bad magic number\n", o->name);

